Gopkg.lock

```diff
@@ -15,7 +15,7 @@
   name = "github.com/btcsuite/btcd"
```

Comment:
can you exclude the updates from this PR?

Reply:
what updates?

Reply:
![fix_errors_in_makefile_by_b00f_ _pull_request__2146_ _tendermint_tendermint_](https://user-images.githubusercontent.com/1282182/43674413-d2861e54-97e4-11e8-892a-e1edf9ce7823.png)

Reply:
I had error on running `dep ensure`. Removing `Gopkg.lock` and running the `dep ensure` fixed the errors.